ImageSegmentation/ImageSegmentation/UIImageHelper.mm

```diff
@@ -0,0 +1,98 @@
+#import <UIKit/UIKit.h>
+
+@implementation UIImageHelper : NSObject
+
+UIImageView* m_imageView;
```

Comment:
Can we define UIImageView*  in UIViewController. This looks uncomfortable to me. You can do something like 

```
// in ViewController.swift
self.imageView.image = UIImageHelper. convertRGBBufferToUIImage(buffer)
```

Reply:
Good catch. Actually the lines 
```
UIImageView* m_imageView;

- (void)setImageView:(UIImageView*)imageView {
    m_imageView = imageView;
}
``` 
in UIImageHelper.mm should be deleted as I already called `self.imageView.image = self.imageHelper.convertRGBBuffer` in ViewController.swift (in main DispatchQueue).